Assets/Scripts/Entity/Entity.cs

```diff
@@ -19,6 +20,13 @@ public abstract class Entity : MonoBehaviour
     // Generic reference to whatever weapon the entity uses for firing
     protected IWeapon fireableWeapon;
 
+    public void setisDead(){
```

Comment:
This isn't really a big deal since it doesn't affect functionality, but I think it's good to be consistent with naming and use CamelCase for functions (i.e., `SetIsDead`). Also, I'd maybe consider renaming this function to `MarkAsDead`, `PlayDead`, or something along those lines, since there's no `isDead` property that's being set (the name's a bit unintuitive, basically).

Reply:
Changed to SetAsDead() :)

Reply:
Sweet, thanks :)